### PR TITLE
Move python code and doc for the vespa API

### PR DESCRIPTION
**Pull request by thigm85**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: python/vespa/.gitignore [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+docs_src/.last_checked
+
+# symlinks to fastai
+docs_src/fastai
+tools/fastai
+
+# link checker
+checklink/cookies.txt
+
+# .gitconfig is now autogenerated
```
Comment (aressem) [inline python/vespa/.gitignore:139]: If this one is autogenerated, it should not be checked in ?

File: python/vespa/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (aressem) [inline python/vespa/LICENSE:1]: No need for this one here it is already in the root of the repo.

File: python/vespa/docs/_config.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+repository: thigm85/vespa
```
Comment (aressem) [inline python/vespa/docs/_config.yml:1]: Its not there anymore

File: python/vespa/docs/_config.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+output: web
+topnav_title: vespa
+site_title: vespa
+company_name: TGM
```
Comment (aressem) [inline python/vespa/docs/_config.yml:5]: Verizon Media ?

File: python/vespa/docs/_data/topnav.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+topnav:
+- title: Topnav
+  items:
+    - title: github
+      external_url: https://github.com/thigm85/vespa/tree/master/
```
Comment (aressem) [inline python/vespa/docs/_data/topnav.yml:5]: Update URL

File: python/vespa/setup.py [hunk 6]
```diff
@@ -0,0 +1,47 @@
+from pkg_resources import parse_version
```
Comment (aressem) [inline python/vespa/setup.py:1]: Missing copyright header in all files.

File: python/vespa/vespa/_nbdev.py [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# AUTOGENERATED BY NBDEV! DO NOT EDIT!
+
+__all__ = ["index", "modules", "custom_doc_links", "git_url"]
+
+index = {}
+
+modules = []
+
+doc_url = "https://thigm85.github.io/vespa/"
```
Comment (aressem) [inline python/vespa/vespa/_nbdev.py:9]: Fix URL

File: python/vespa/vespa/_nbdev.py [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+git_url = "https://github.com/thigm85/vespa/tree/master/"
```
Comment (aressem) [inline python/vespa/vespa/_nbdev.py:11]: Fix URL

File: python/vespa/settings.ini [hunk 9]
```diff
@@ -0,0 +1,56 @@
+[DEFAULT]
+# All sections below are required unless otherwise specified
+host = github
+lib_name = vespa
+user = thigm85
+description = A description of your project
+keywords = vespa, search engine, data science
+author = Thiago G. Martins
+author_email = thigm85@gmail.com
+copyright = TGM
```
Comment (aressem) [inline python/vespa/settings.ini:10]: Verizon Media

**Comment by aressem** [comment]
I have briefly scanned through the python code, but as this is work in progress I have not done a thorough review.

**Review by aressem — CHANGES_REQUESTED** [review]
Se comments in review.

File: python/vespa/.gitignore (hunk 1 above)
Comment (thigm85) [reply python/vespa/.gitignore:139]: It came with the package template. Is there an issue to have it within the python package folder?

**Review by aressem — APPROVED** [review]

**Pull request merged**